AfricanReligions/localisation/afr_general_localization.csv

```diff
@@ -324,4 +330,6 @@ secret_religious_society_amazigh_reformed_rank_2_female;Adherent;;;;;;;;;;;;;x
 secret_religious_society_amazigh_reformed_rank_3_male;Preacher;;;;;;;;;;;;;x
 secret_religious_society_amazigh_reformed_rank_3_female;Preacher;;;;;;;;;;;;;x
 secret_religious_society_amazigh_reformed_rank_4_male;Herald;;;;;;;;;;;;;x
-secret_religious_society_amazigh_reformed_rank_4_female;Herald;;;;;;;;;;;;;x
\ No newline at end of file
+secret_religious_society_amazigh_reformed_rank_4_female;Herald;;;;;;;;;;;;;x
+pick_religious_tradition;Choose a Religious Tradition;;;;;;;;;;;;;x
+pick_religious_tradition_tooltip;Choose a religious tradition to follow.;;;;;;;;;;;;;x
```

Comment:
This file is actually organized - there are sections for decision localizations (and you should use the description from the A Tat Roog decisions spreadsheet, although I think this name is fine) ending after line 220 and a place for tooltip localizations ending after line 195.  If you add more tooltips to the options as I suggested you should put their localizations there as well.